File: ws/src/main/scala/de/thm/ii/fbs/services/checker/trait/CheckerServiceFormatSubmission.scala
package de.thm.ii.fbs.services.checker.`trait`

import de.thm.ii.fbs.model.{CheckrunnerConfiguration, Submission}

trait CheckerServiceFormatSubmission {
  def formatSubmission(submissionID: Submission, checkrunnerConfiguration: CheckrunnerConfiguration, solution: String): Any
}

File: modules/fbs-core/api/src/main/kotlin/de/thm/ii/fbs/services/v2/checker/trait/CheckerServiceFormatSubmission.kt
package de.thm.ii.fbs.services.v2.checker.trait

import de.thm.ii.fbs.model.v2.CheckrunnerConfiguration
import de.thm.ii.fbs.model.v2.Submission

interface CheckerServiceFormatSubmission {
    fun formatSubmission(submissionID: Submission, checkrunnerConfiguration: CheckrunnerConfiguration, solution: String): Any
}